miniscope_io/stream_daq.py

```diff
@@ -163,7 +163,9 @@ def _parse_header(self, buffer: bytes) -> Tuple[StreamBufferHeader, np.ndarray]:
             reverse_payload_bytes=self.config.reverse_payload_bytes,
         )
 
-        header_data = StreamBufferHeader.from_format(header, self.header_fmt, construct=True)
+        header_data = StreamBufferHeader.from_format(
+            header.astype(int), self.header_fmt, construct=True
+        )
```

Comment:
Probably not a big problem but just a note so I'll come back later.

I did this because I saw these Pydantic warnings:
```
tests/test_stream_daq.py::test_csv_output[buffer-size-5-True]
tests/test_stream_daq.py::test_csv_output[buffer-size-10-True]
  C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\site-packages\pydantic\main.py:389: UserWarning: Pydantic serializer warnings:
    Expected `int` but got `uint32` with value `np.uint32(29)` - serialized value may not be as expected
    Expected `int` but got `uint32` with value `np.uint32(12939)` - serialized value may not be as expected
    Expected `int` but got `uint32` with value `np.uint32(103517)` - serialized value may not be as expected
    Expected `int` but got `uint32` with value `np.uint32(5)` - serialized value may not be as expected
    Expected `int` but got `uint32` with value `np.uint32(103515)` - serialized value may not be as expected
    Expected `int` but got `uint32` with value `np.uint32(0)` - serialized value may not be as expected
    Expected `int` but got `uint32` with value `np.uint32(629827)` - serialized value may not be as expected
    Expected `int` but got `uint32` with value `np.uint32(0)` - serialized value may not be as expected
    Expected `int` but got `uint32` with value `np.uint32(5072)` - serialized value may not be as expected
    Expected `int` but got `uint32` with value `np.uint32(0)` - serialized value may not be as expected
    Expected `int` but got `uint32` with value `np.uint32(0)` - serialized value may not be as expected
    return self.__pydantic_serializer__.to_python(
```

But I'm now getting this:
```
tests/test_stream_daq.py::test_csv_output[buffer-size-5-True]
tests/test_stream_daq.py::test_csv_output[buffer-size-10-True]
  /opt/hostedtoolcache/Python/3.9.19/x64/lib/python3.9/site-packages/pydantic/main.py:389: UserWarning: Pydantic serializer warnings:
    Expected `int` but got `int64` with value `np.int64(26)` - serialized value may not be as expected
    Expected `int` but got `int64` with value `np.int64(12827)` - serialized value may not be as expected
    Expected `int` but got `int64` with value `np.int64(102618)` - serialized value may not be as expected
    Expected `int` but got `int64` with value `np.int64(2)` - serialized value may not be as expected
    Expected `int` but got `int64` with value `np.int64(102617)` - serialized value may not be as expected
    Expected `int` but got `int64` with value `np.int64(0)` - serialized value may not be as expected
    Expected `int` but got `int64` with value `np.int64(624369)` - serialized value may not be as expected
    Expected `int` but got `int64` with value `np.int64(0)` - serialized value may not be as expected
    Expected `int` but got `int64` with value `np.int64(5072)` - serialized value may not be as expected
    Expected `int` but got `int64` with value `np.int64(0)` - serialized value may not be as expected
    Expected `int` but got `int64` with value `np.int64(0)` - serialized value may not be as expected
    return self.__pydantic_serializer__.to_python(
```

Reply:
oh yeah i have been meaning to handle this, super annoying. re: https://github.com/pydantic/pydantic/issues/9677

i've got this one

Reply:
Sounds great! Not critical atm but hope you guys resolve it.